api/v1alpha1/etcd.go

```diff
@@ -233,6 +233,22 @@ type EtcdConfig struct {
 	// ClientService defines the parameters of the client service that a user can specify
 	// +optional
 	ClientService *ClientService `json:"clientService,omitempty"`
+	// ExistingCluster specifies the configuration for an existing etcd cluster that the etcd-druid managed cluster will join.
+	// +optional
+	ExistingCluster *ExistingCluster `json:"existingCluster,omitempty"`
+}
+
+// ExistingCluster defines the configuration for an existing etcd cluster.
+type ExistingCluster struct {
+	// MemberURLs specifies the URLs of the existing etcd cluster members.
```

Comment:
The docstring does not give sufficient information about the structure of the `MemberURLs` field. What do the keys represent? Member names? Member IDs? Quite unclear.

Also, I feel it would be cleaner to rename `MemberURLs` to `Members`, which is a slice of `ExistingClusterMember` type.

```
type ExistingClusterMember struct {
    Name string
    ID string
    URLs []string
}
```

Something on these lines, to make the consumption clear and explicit. Using the map key to denote member name/ID becomes confusing for the exact reason that I mentioned above, that it isn't intrinsically clear whether the key represents member name or ID.